x/blob/types/blob_tx.go

```diff
@@ -15,7 +15,7 @@ type Blob = tmproto.Blob
 
 // NewBlob creates a new coretypes.Blob from the provided data after performing
 // basic stateless checks over it.
-func NewBlob(ns appns.Namespace, blob []byte) (*Blob, error) {
+func NewBlob(ns appns.Namespace, blob []byte, share uint8) (*Blob, error) {
```

Comment:
the name `share` here for the share version is a bit confusing imo. Also, we should pass it to the blob below

Reply:
changed to shareVersion, done.